easybuild/easyconfigs/f/FDS/FDS-6.9.1-foss-2023b.eb

```
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'FDS'
+version = '6.9.1'
+
+homepage = 'https://pages.nist.gov/fds-smv'
+description = """Fire Dynamics Simulator (FDS) is a large-eddy simulation (LES) code for low-speed flows,
+ with an emphasis on smoke and heat transport from fires."""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'pic': True, 'usempi': True, 'openmp': True}
+
+source_urls = ['https://github.com/firemodels/fds/archive/']
+sources = ['FDS-%(version)s.tar.gz']
+checksums = ['e0fe22d1386dc06512489ddf190dbdbee4c9865856f4a7ca48aec8425b4c0867']
+
+unpack_options = '--strip-components=1'
+
+start_dir = 'Build'
+
+# just run make in the install dir
+skipsteps = ['configure', 'install']
+buildininstalldir = True
+
+buildopts = 'ompi_gnu_linux &&'
+buildopts += 'cd %(installdir)s/Build && ln -s fds_ompi_gnu_linux fds'
+
+modextrapaths = {'PATH': 'Build'}
+
+sanity_check_paths = {
+    'files': ['Build/fds'],
+    'dirs': [],
+}
+
+sanity_check_commands = [
+    "fds -v",
```

existing easyconfigs for FDS 6.9.1 use:
```suggestion
    "fds 2>&1 | grep 'MPI version'",
```